### ISSUE TITLE
Missing file reference in error message

### ISSUE THREAD

**Issue by MarcinOrlowski** [johncarl81/parceler#156]
Can error message like this:

    Error:Parceler: No @ParcelConstructor annotated constructor and no default empty bean constructor found.

contain name of the file/class this error is related to?

**Comment by johncarl81** [comment]
It should reference the class there... what build tool are you using?

**Comment by johncarl81** [comment]
I've confirmed this is a bug.  Here's what the output should looks like:

Gradle:
```
ExampleParcel.java:24: error: Parceler: No @ParcelConstructor annotated constructor and no default empty bean constructor found.
public class ExampleParcel {
       ^
```
Maven:
```
ExampleParcel.java:[25,8] Parceler: No @ParcelConstructor annotated constructor and no default empty bean constructor found.
```

And here's the fix: 36dd4e906d6913da15b9a06a891c0568333df64c (https://github.com/johncarl81/transfuse/commit/b3f4b18926336d0c4b520814e1ccdd701d8fd894)

**Comment by MarcinOrlowski** [comment]
Thanks for quick fix!
